python/pip.bzl

```diff
@@ -56,6 +56,17 @@ def pip_install(requirements, name = "pip", **kwargs):
         **kwargs
     )
 
+def pip_install_incremental(requirements_lock, name = "pip_incremental", **kwargs):
```

Comment:
Why is this a separate macro and not just a conditional on the existing `pip_install` macro?

Reply:
That macro itself gives a good example of how to do it.